example/src/App.js

```diff
@@ -9,29 +9,20 @@ import './App.css';
 import '@criipto/verify-react/dist/criipto-verify-react.css';
 
 function App() {
-  const { claims, isLoading } = useCriiptoVerify();
+  const { claims, isLoading, logout } = useCriiptoVerify();
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [currentUser, setCurrentUser] = useState({});
```

Comment:
These two state values are probably not required, the `claims` value can be used to determine both.

Reply:
Removed both, using `claims` now.